cluster/examples/composition/azure-sqlserver/definitions/networkgroup.yaml

```diff
@@ -0,0 +1,155 @@
+---
+# This allows Crossplane to manage our new "NetworkGroup" composite resource and
+# the resources it composes.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: networkgroups.azure.example.org
+  labels:
+    rbac.crossplane.io/aggregate-to-crossplane: "true"
+rules:
+- apiGroups:
+  - azure.example.org
+  resources:
+  - networkgroups
+  - networkgroups/status
+  verbs:
+  - "*"
+- apiGroups:
```

Comment:
You don't need to grant permissions for azure stuff here since stack installation already covers them.